src/type-extensions.ts

```diff
@@ -126,11 +127,38 @@ declare module "hardhat/types/runtime" {
             getWallet: (name: string) => WalletConfig;
 
             devnet: Devnet;
+
+            /**
+             * Deploys an Account contract based on the ABI and the type of Account selected
+             * @param accountContract the case-sensitive contract name, same as {@link getContractFactory}
+             * @param accountType the enumerator value of the type of Account to use
+             * @returns an Account object
+             */
+            deployAccountFromABI: (
```

Comment:
Will not be needed if we decide to expose OpenZeppelinAccount class with its constructor.